src/main.rs

```diff
@@ -782,14 +783,40 @@ struct Cli {
     /// Used by child process such that supervisor parent process can control it
     #[arg(long)]
     child_process: bool,
+    /// Show uninstall dialog and delete configuration and logs
+    #[arg(long)]
+    uninstall: bool,
     /// The rest of the arguments that will be sent to GTK4 as is
     #[arg(raw = true)]
     gtk_arguments: Vec<String>,
 }
 
 impl Cli {
     fn run(self) -> ExitCode {
-        if self.child_process {
+        if self.uninstall {
+            if cfg!(windows)
+                && MessageDialog::new()
+                    .set_type(MessageType::Info)
+                    .set_title("Uninstall")
+                    .set_text("Delete Space Acres configuration and logs for all users?")
+                    .show_confirm()
+                    .is_ok_and(|x| x)
```

Comment:
I thought, it would be better to avoid GTK4 "bootstrapping" during uninstallation to prevent freezing and flickering.
It seems, these suggestions are quite beyond the requirements of the original issue. :(

Reply:
> I thought, it would be better to avoid GTK4 "bootstrapping" during uninstallation to prevent freezing and flickering

Not sure what you're talking about here, can you elaborate.

> It seems, these suggestions are quite beyond the requirements of the original issue. :(

Yes and no. It will work as is for sure, but original issue expected this to be handled in the wix itself, now that you found a nice workaround, it wasn't expected that in the app that has a GUI toolkit already we'd be pulling a bunch of additional GUI dependencies for a simple purpose to (initially) display a confirmation dialog.

I'll do it myself if too difficult, no worries. But I don't expect us keeping `native-dialog` in dependencies.

Reply:
> Not sure what you're talking about here, can you elaborate.
GTK4 GUI causes noticeable delay starting up on low end systems, it also has some display artifacts at least on VMs.
Also, system native dialog conforms to the installer interface much better.